include/text.php

```diff
@@ -1975,17 +1977,22 @@ function protect_sprintf($s) {
 	return str_replace('%', '%%', $s);
 }
 
-
+/// @TODO Rewrite this
```

Comment:
Can you be more specific, what needs rewriting? Is it not working as intended?

Reply:
Strange, no more in my source code? :confused:

Reply:
Still is in the PR though.

Reply:
I have removed it, I don't remember that I want to rewrite there.